options/locale/locale_bg-BG.ini

```diff
@@ -365,8 +365,8 @@ pulls.new=Нова заявка за сливане
 pulls.filter_branch=Филтър по клон
 pulls.no_results=Няма резултати.
 pulls.create=Създай заявка за сливане
-pulls.title_desc=заяви обединяване на %[1]d ревизии от <code>%[2]s</code> във <code>%[3]s</code>
-pulls.merged_title_desc=обедини %[1]d ревизии от <code>%[2]s</code> във <code>%[3]s</code> %[4]s
+pulls.title_desc=заяви обединяване на %[1]d ревизии от %[2]s във %[3]s
+pulls.merged_title_desc=обедини %[1]d ревизии от %[2]s във %[3]s %[4]s
```

Comment:
Please reset changes to translations as they need to be updated in crowding.com and will be reset on merge anyway (except for English)

Reply:
Gotcha. Wasn't sure as to the mechanism by which that took place. Will do that.